### DISCUSSION TITLE
Is there a support for Minecraft 1.19?

### DISCUSSION THREAD

**Discussion by popos123** [IoeCmcomc/MCDatapacker#5] [Q&A] [answered]
Hello, like in the title, whats the plan for 1.19?

**Comment by IoeCmcomc** [comment, marked as answer]
There are some plans for newer Minecraft versions. However, I'm currently quite busy so the progress will be slow. Adding new command syntaxes is easy, but adding other features (i.e. loot table generator) requires some works due to their complexity.
